### DISCUSSION TITLE
How to set the connected edge to be circular when the source node and target node are the same?

### DISCUSSION THREAD

**Discussion by Schwarao** [xyflow/xyflow#3606] [Q&A]
When my user uses Reactflow to create nodes and edges, if the connected source and target nodes are the same when creating edges, the created edges are circular, that is, the graph bypassing nodes (not above or covered by nodes)

**Comment by bcakmakoglu** [comment]
You can achieve something like that using a custom edge.

[Here's an example.](https://codesandbox.io/s/sharp-orla-vyjqen?file=/src/CustomEdge.tsx)

**Reply by Schwarao** [reply]
But this is a fixed edge, and I hope users will generate this semi circular edge when connecting the source and target nodes of the same node
